build.gradle

```diff
@@ -170,6 +187,15 @@ dependencies {
 
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
+
+  smokeTestImplementation sourceSets.main.runtimeClasspath
+  smokeTestImplementation sourceSets.test.runtimeClasspath
```

Comment:
this is not needed. smoke test should not depend on overkill of spring test framework unless absolutely necessary